jobs/tests.py

```diff
@@ -16,77 +14,105 @@ class JobModelTests(TestCase):
 
     def setUp(self):
         """Setting up models with mommy."""
-        self.job = mommy.make(Job, ready_to_publish=True)
-        self.job1 = mommy.make(Job, ready_to_publish=False)
-
-    def test_publish_succesfully(self):
-        """Plain publish method test without any twists."""
-        self.assertFalse(self.job.published_date)
-        self.job.publish()
-        self.assertTrue(self.job.published_date, "Job has no published date.")
-
-    def test_publish_unsuccesfully(self):
-        """Publish doesn't work when ready_to_publish field is set to False."""
-        self.assertFalse(self.job1.published_date)
-        self.job1.publish()
-        self.assertFalse(self.job1.published_date, "Job has published date.")
-
-    def test_publish_more_than_once_and_change_values(self):
-        """Publish once, than change ready_to_publish field value,
-        publish once more."""
-        self.assertFalse(self.job.published_date)
-        self.job.publish()
-        self.job.ready_to_publish = False
-        # Right now if we change ready_to_publish to False,
-        # published_date stays as it was and doesn't become None.
-        self.assertTrue(self.job.published_date)
-        self.job.ready_to_publish = True
-        self.job.publish()
-        self.assertTrue(self.job.published_date, "Job has no published date.")
+        self.custom_date = datetime(2016, 2, 14, 15, 30)
```

Comment:
The warnings were there mostly because of migrations. Some of them set the default for the expiration date which was done via naive datetime. I've removed those defaults from migrations (will push in a sec). Maybe we should clean the migrations up a bit...